SetupDependencies.ps1

```diff
@@ -25,6 +33,6 @@ If ($env:CI -eq $false)
     ElseIf ($IsLinux) { npm install -g dotnet-sdk-3.1-linux-x64 }
     Else { Write-Error ('Unknown Operation System') }
 }
-npm install -g autorest@latest
+npm install -g autorest@3.4.2
```

Comment:
Probably not a bad idea to pin the version but was there a reason that made you want to do this?

Reply:
I found it rather difficult to help troubleshoot when versions of autorest core/ powershell were not consistent. The headers of generated files did change in the new version which prompted several changes in this PR. 

I think we are find to keep it at a specific version and test new versions when an autorest feature worth upgrading is released.